docs/services/firecrest.md

```diff
@@ -1,37 +1,86 @@
+
+<table>
+<tr><th>Platform</th><th>Version</th><th>API Endpoint</th><th>Clusters</th></tr>
+<tr><td style="vertical-align: middle;" rowspan="2">HPC Platform</td><td>v1</td><td>https://api.cscs.ch/hpc/firecrest/v1</td><td style="vertical-align: middle;" rowspan="2"><a href="../../clusters/daint">Daint</a>, <a href="../../clusters/eiger">Eiger</a></td></tr>
+<tr>                                 <td>v2</td><td>https://api.cscs.ch/hpc/firecrest/v2</td></tr>
+<tr><td>ML Platform</td><td>v1</td><td>https://api.cscs.ch/ml/firecrest/v1</td><td style="vertical-align: middle;"><a href="../../clusters/bristen">Bristen</a>, <a href="../../clusters/clariden">Clariden</a></td></tr>
+<tr><td style="vertical-align: middle;" rowspan="2">CW Platform</td><td>v1</td><td>https://api.cscs.ch/cw/firecrest/v1</td><td style="vertical-align: middle;" rowspan="2"><a href="../../clusters/santis">Santis</a></td></tr>
+<tr><td>v2</td><td>https://api.cscs.ch/cw/firecrest/v2</td></tr>
+</table>
+
 
-| Platform | API Endpoint | Clusters |
-|----------|-------------|-----------|
-| HPC Platform | https://api.cscs.ch/hpc/firecrest/v1 | [Daint][ref-cluster-daint], [Eiger][ref-cluster-eiger] |
-| ML Platform | https://api.cscs.ch/ml/firecrest/v1 | [Bristen][ref-cluster-bristen], [Clariden][ref-cluster-clariden] |
-| CW Platform | https://api.cscs.ch/cw/firecrest/v1 | [Santis][ref-cluster-santis] |
+## Accessing FirecREST
 
-## Developer Portal: Creating FirecREST clients
+### Clients and Access tokens
 
-The [Developer Portal](https://developer.cscs.ch) is an application based on [WSO2 API Manager](https://wso2.com/api-manager/) that facilitates CSCS Users to manage subscriptions to an API at CSCS (such as FirecREST).
+A **client application** that makes requests to FirecREST will not be using directly the credentials of the user for the authentication, but an **access token** instead. The access token is a signed JSON Web Token ([JWT](https://jwt.io/introduction)) which contains user information and it's valid for a short time (5 minutes). Behind the API, all commands launched by the client will use the account of the user that registered the client, inheriting their access rights.
 
-Start by navigating to [developer.cscs.ch](https://developer.cscs.ch), then sign in by clicking the "SIGN-IN" button on the top right hand corner of the page.
+Every client will have a client ID (Consumer Key) and a secret (Consumer Secret) that will be used to get a short-lived access token with an HTTP request.
+
+!!! example "curl call to fetch the access token"
+    ```
+    curl -s -X POST https://auth.cscs.ch/auth/realms/firecrest-clients/protocol/openid-connect/token \
+         --data "grant_type=client_credentials" \
+         --data "client_id=<your_client>" \
+         --data "client_secret=<your_secret>"
+    ```
+
+You can manage your client application on the [CSCS Developer Portal][ref-devportal].
+
+[](){#ref-devportal}
```

Comment:
To make the links inside the page a little bit "more unique" I would suggest adding and extra `-f7t` between `ref` and  the sub-heading:
`ref-f7t-devportal`

Reply:
Well, the Developer Portal is not specific to FirecREST. For instance, CI-Ext is using it for external users.